modules/virt-cloning-vm-snapshot.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-backup-restore-snapshots.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="virt-cloning-vm-snapshot_{context}"]
+= Cloning a snapshot to create a VM by using the web console
+
+You can clone an existing virtual machine (VM) snapshot to create a new VM.
```

Comment:
You can clone an existing virtual machine (VM) to a new VM, if VM's underlying storage supports snapshots.